### PR TITLE
add uint8 support for squeeze operator

### PR DESCRIPTION
**Pull request by Joejiong**
### PR types

Bug fixes
### PR changes

OPs 
### Describe

Adding uint8 support for squeeze operator.

Paddle user raised the following error:
unsqueeze, reshape and several other operators do not support uint8 type tensor:

image

After fixing:
image

### CODE REVIEW

File: python/paddle/fluid/tests/unittests/test_squeeze_op.py [hunk 1]
```diff
@@ -113,7 +116,25 @@ class API_TestDygraphSqueeze(unittest.TestCase):
...
             input_1 = np.random.random([5, 1, 10]).astype("int32")
-            input = fluid.dygraph.to_variable(input_1)
+            input = paddle.to_tensor(input_1)
+            output = paddle.squeeze(input, axis=[1])
+            out_np = output.numpy()
+            expected_out = np.squeeze(input_1, axis=1)
+            self.assertTrue(np.allclose(expected_out, out_np))
+
+    def test_out_int8(self):
+        with fluid.dygraph.guard():
```
Comment (zhhsplendid) [inline python/paddle/fluid/tests/unittests/test_squeeze_op.py]: 这里要改paddle.static.program_guard吗？

File: python/paddle/fluid/tests/unittests/test_squeeze_op.py [hunk 2]
```diff
@@ -113,7 +116,25 @@ class API_TestDygraphSqueeze(unittest.TestCase):
...
+            input_1 = np.random.random([5, 1, 10]).astype("int8")
+            input = paddle.to_tensor(input_1)
+            output = paddle.squeeze(input, axis=[1])
+            out_np = output.numpy()
+            expected_out = np.squeeze(input_1, axis=1)
+            self.assertTrue(np.allclose(expected_out, out_np))
+
+    def test_out_uint8(self):
+        with fluid.dygraph.guard():
```
Comment (zhhsplendid) [inline python/paddle/fluid/tests/unittests/test_squeeze_op.py]: 这里要改paddle.static.program_guard吗？

File: python/paddle/fluid/tests/unittests/test_unsqueeze_op.py [hunk 3]
```diff
@@ -183,14 +189,61 @@ def test_out(self):
...
+                paddle.to_tensor(input1),
                 axis=[paddle.to_tensor([1]), paddle.to_tensor([2])])
             out_np = output.numpy()
             self.assertTrue(np.array_equal(out1, out_np))
             self.assertEqual(out1.shape, out_np.shape)
 
 
+class API_TestDygraphUnSqueeze(unittest.TestCase):
+    def test_out(self):
+        with fluid.dygraph.guard():
```
Comment (zhhsplendid) [inline python/paddle/fluid/tests/unittests/test_unsqueeze_op.py]: 这里要改paddle.static.program_guard吗？下面几个类似的地方问题相同

File: python/paddle/fluid/tests/unittests/test_squeeze_op.py (hunk 1 above)
Comment (Joejiong) [reply python/paddle/fluid/tests/unittests/test_squeeze_op.py]: 已替换为enable_static和disable_static，done

File: python/paddle/fluid/tests/unittests/test_squeeze_op.py (hunk 2 above)
Comment (Joejiong) [reply python/paddle/fluid/tests/unittests/test_squeeze_op.py]: 已替换为enable_static和disable_static，done

File: python/paddle/fluid/tests/unittests/test_unsqueeze_op.py (hunk 3 above)
Comment (Joejiong) [reply python/paddle/fluid/tests/unittests/test_unsqueeze_op.py]: 已替换为enable_static和disable_static，done

**Review by zhhsplendid — APPROVED** [review]
LGTM

**Pull request merged**